unfetter-discover/unfetter-ui
# fixes heatmap and treemap displays of incomplete tactics data

Fixes unfetter-discover/unfetter#1122.

Steps to test:
- As in the ticket above, create an attack pattern in Mongo for a framework that has no current attack pattern data (such as mitre-mobile-attack). Can have just a name and kill_chain information, nothing detailed.
- Bring up the developer console to keep watch for any errors.
- In the UI, change your settings to the framework you used above.
- Bring up the Analytics Exchange.
    - The heatmap should display all the phases, with no data in their columns, save for the one tactic you added.
    - The treemap should also display all the phases.
- Change your settings to a framework with no tactics loaded at all (possibly mitre-pre-attack or lockheed-martin).
- Return to the Analytics Exchange.
    - Again the heatmap should list all the phases, with all of the columns empty.
    - The treemap should again display all the phases.
- Developer console should be free of errors.
